inst/rmarkdown/templates/gigascience_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,548 @@
+reftype: num-refs
+#reftype: alpha-refs
+
+# pandoc_args: arguments to pandoc
+#   - enable with '+' and disable with '-'
+#       - see: https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html#pandoc_markdown
+#   - concerning specific keys:
+#       raw_tex:
+#         - allow direct inclusion of TeX
+#         - see https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html#raw-tex`
+pandoc_args: +raw_tex
+
+# Using rmarkdown for citations is not recommended: 
+#   - TeX citations give you more flexibility
+#   - e.g., references can be deleted or added subsequent to translation from rmarkdown to TeX
+# bibliography and csl are only needed for citations from rmarkdown
+# bibliography: the file with the bibtex entries
+#bibliography: paper-refs.bib
+# csl: this is needed only for citations from rmarkdown 
+#csl: gigascience.csl
+
+# output: specifies the R class used to translate rmarkdown to tex and pdf
+output: rticles::gigascience_article
+---
+
+\begin{epigraph}  {\small{\href{https://academic.oup.com/gigascience/pages/instructions_to_authors}{GigaScience Instructios to Authors}}}
+Sample epigraph: For creating manuscripts in LaTeX, Gigascience recommends the use of its own LaTeX class files. Our class files are available online \href{https://www.overleaf.com/latex/templates/template-for-gigascience-journal-manuscript-submissions/shgtrssvbjhs}{at Overleaf https://www.overleaf.com/latex/templates/template-for-gigascience-journal-manuscript-submissions/shgtrssvbjhs}.  The GigaScience "oup-contemporary.cls" class obtained via this Overleaf link was used as the basis of this "rticle".
+\end{epigraph}
+
+TeX/LaTeX users: Please use OUP’s TeX template and BibTeX stylefile if you use TeX format. During the TeX submission process, please submit your TeX file as the main manuscript file and your bib/bbl file as a dependent file. Please also convert your TeX file into a PDF and submit this PDF as an additional file with the name 'Reference PDF'. This PDF will be used by internal staff as a reference point to check the layout of the article as the author intended. If you have used another template for your manuscript, or if you do not wish to use BibTeX, then please submit your manuscript as a DVI file. We do not recommend converting to RTF.  For all TeX submissions, all relevant editable source must be submitted during the submission process. Failing to submit these source files will cause unnecessary delays in the publication procedures.  [https://academic.oup.com/gigascience/pages/technical_note](https://academic.oup.com/gigascience/pages/technical_note) 
+
+# Notes for using this `rticle` template
+
+## About the GigaScience `rticle` template
+
+The `rticles::gigascience_journal` template is the blended `rmardkdown` and \LaTeX{} template for GigaScience journal manuscript submissions.  If you do not want to include \LaTeX{} commands inline, remove the following line from the YAML header.
```

Comment:
I guess it will be extremely unlikely that an author will not need the `raw_tex` extension, so I'd just not mention this at all.